app/konnect/updates.md

```diff
@@ -10,6 +10,11 @@ from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 
+## November 2023
+
+**Support for exporting configurations in YAML and JSON**
+: {{site.konnect_short_name}} users can now export Gateway Manager objects like services, routes, plugins to YAML or JSON and import them into APIOps tooling like DecK.
```

Comment:
```suggestion
: {{site.konnect_short_name}} users can now export Gateway Manager objects like services, routes, plugins to YAML or JSON and manage them with APIOps tooling like decK.
```
"Import them into decK" is not really a sentence that makes sense. You can't really import things into decK, it's a CLI tool that imports/exports stuff for you.